test/functional/fixtures/api/es-next/navigate-to/test.js

```diff
@@ -0,0 +1,16 @@
+var expect = require('chai').expect;
+
+
+describe('[API] t.navigateTo', function () {
+    it('Should navigate to the specified url', function () {
+        return runTests('./testcafe-fixtures/navigate-to-test.js', 'Navigate to another page');
+    });
+
+    it('Should validate the url argument', function () {
+        return runTests('./testcafe-fixtures/navigate-to-test.js', 'Incorrect protocol', { shouldFail: 'true' })
+            .catch(function (errs) {
+                expect(errs[0]).contains('The url argument specifies a URL that uses an unsupported ftp protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.');
```

Comment:
Too bad we don't have `://` after protocol name in error message. Can we add this functionality in context of this PR?